src-docs/src/views/header/header.js

```diff
@@ -41,7 +37,7 @@ import { htmlIdGenerator } from '../../../../src/services';
 export default () => {
   const renderLogo = () => (
     <OuiHeaderLogo
-      iconType="logoElastic"
+      iconType="logoOUI"
```

Comment:
Should these be `logoOpensearch` instead?

As per https://github.com/opensearch-project/oui/issues/111#issuecomment-1397891063

Reply:
I think so - originally I think there was a miscommunication about the replacement of Elastic to OUI and got swept-up in find>replace instances. I do believe we should be making this logoOpensearch instead, as the logoOUI "namespace" should be used for the actual OUI logo.

Reply:
@KrooshalUX I think this one is very close to merge. Could you fix this comment?